lib/cartopy/tests/test_geodesic.py

```diff
@@ -0,0 +1,54 @@
+# (C) British Crown Copyright 2015, Met Office
+#
+# This file is part of cartopy.
+#
+# cartopy is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# cartopy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with cartopy.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import (absolute_import, division, print_function)
+
+import unittest
+
+import numpy as np
+from numpy.testing import assert_almost_equal, assert_array_almost_equal
+from nose.tools import assert_equal
+
+from cartopy import geodesic
+
+
+class TestGeodesic(unittest.TestCase):
+
+    def test_dir(self):
+        geod = geodesic.Geodesic()
+        geod_dir = geod.direct([40, 50], 38, 2000)
+        assert_almost_equal(geod_dir, np.array([[40.01717933, 50.01416786,
```

Comment:
I don't think this is right, the backwards azimuth looks wrong. Can you reproduce it with the `geod` command line tool from PROJ4? I tried the following which I thought should be the same but it isn't, but then I'm not very familiar with the PROJ4 tools so I may be doing something wrong:

```
geod -f '%.16f' +a=6378137.0 +f=0.0033528106647474805 <<EOF +units=m
40 50 38 2000
EOF

40.0141567459005856	50.0144423164747636	-141.9907152910965067
```

But then again with pyproj I get the solution:

```(40.017179329102426, 50.01416786311794, -141.98683850507194)```

The end point agrees with yours but not the back azimuth.